### ISSUE TITLE
Getting issue internal server error on client.rest.withdraw.withdrawToCryptoAddress, please help

### ISSUE THREAD

**Issue by Gauravsanganan** [bennycode/coinbase-pro-node#609]
Getting issue internal server error on client.rest.withdraw.withdrawToCryptoAddress, please help

**Comment by hjoelr** [comment]
@Gauravsanganan I'm not the repo maintainer. However, I think some more information would be helpful. What currency are you using? Can you give a code snippet showing how you're using the function? Also, can you look at https://status.coinbase.com/ and see if there might have been an issue on Coinbase's side when you were trying to use the function?

**Comment by Gauravsanganan** [comment]
@bennycode please help me on this, bit urgent

**Comment by bennycode** [comment]
@Gauravsanganan your execution looks good to me. Are you sure that your client is connected to the production environment of Coinbase and not to it's Sandbox? Do you get an other message than "Request failed with status code 500"? Can you make the transfer (0.1 ETH to "0xB67FECF7...") manually?

**Comment by hjoelr** [comment]
@Gauravsanganan Another thing to check is if you have address whitelisting enabled at https://pro.coinbase.com/profile/address-book and the address you're attempting to send to is not whitelisted.

**Comment by Gauravsanganan** [comment]
@hjoelr @bennycode thanks for quick response, I am using sandbox environment, not on Production, so withdrawtocrypto only works on production?

**Comment by Gauravsanganan** [comment]
Please respond on this, withdrawtocrypto only works with Production, not on dev env?

**Comment by bennycode** [comment]
To use the withdrawals endpoint it requires the "transfer" permission. Your API key also has to belong to the default profile.

**Comment by Gauravsanganan** [comment]
@bennycode thanks for reply, I am using sandbox keys, and also transfer permission is there in both(prod and sandbox) so this piece of code would work in prod keys(from https://pro.coinbase.com/) not with Sandbox(keys from https://public.sandbox.pro.coinbase.com/)

**Comment by bennycode** [comment]
Can you do the withdrawal in the sandbox when using Coinbase's Web UI?

**Comment by Gauravsanganan** [comment]
@bennycode yes from https://public.sandbox.pro.coinbase.com/ > withdraw > select address & ETH > withdraw > success

**Comment by Gauravsanganan** [comment]
Please let me know if withdraw is possible with Sandbox or not? @bennycode @hjoelr

**Comment by bennycode** [comment]
Can you please send me a screenshot from the balances of your **sandbox portfolio**? https://public.sandbox.pro.coinbase.com/portfolios

**Comment by Gauravsanganan** [comment]
Sure, here it is
![Screen Shot 2021-09-30 at 3 18 44 PM](https://user-images.githubusercontent.com/55246658/135429853-0c8f56cd-4883-48af-8c9a-ffbd9891a694.png)

**Comment by Gauravsanganan** [comment]
@bennycode I can share my keys, if you want to test.

**Comment by bennycode** [comment]
No, please don't share your keys. I will test it with my own account but I won't have time to do it this week.

**Comment by Gauravsanganan** [comment]
Ok please test and let me know asap as bit urgent, work has been stuck, thanks again

**Comment by bennycode** [comment]
Please note that the **coinbase-pro-node** library is a hobby project. I work on it in my spare time. So I won't let myself be rushed in time.

For urgent matters, please contact the Coinbase Pro support: https://help.coinbase.com/en/pro/getting-started/other/how-to-contact-coinbase-pro-support

**Comment by Gauravsanganan** [comment]
@bennycode I contacted them and waiting for their response, meanwhile if you can check. Thanks again

**Comment by Gauravsanganan** [comment]
@bennycode did you get chance to check?

**Comment by bennycode** [comment]
No
